src/firmware_new/boards/frankie_v1/lwipopts.h

```diff
@@ -48,16 +48,26 @@ extern "C"
 
 /* LwIP Stack Parameters (modified compared to initialization value in opt.h) -*/
 /* Parameters set in STM32CubeMX LwIP Configuration GUI -*/
-/*----- Value in opt.h for LWIP_DHCP: 0 -----*/
-#define LWIP_DHCP 1
 /*----- Default Value for LWIP_IGMP: 0 ---*/
 #define LWIP_IGMP 1
+/*----- Default Value for MEM_LIBC_MALLOC: 0 ---*/
+#define MEM_LIBC_MALLOC 1
+/*----- Default Value for MEMP_MEM_MALLOC: 0 ---*/
+#define MEMP_MEM_MALLOC 1
 /*----- Value in opt.h for MEM_ALIGNMENT: 1 -----*/
 #define MEM_ALIGNMENT 4
+/*----- Default Value for MEM_SIZE: 1600 ---*/
+#define MEM_SIZE 32768
```

Comment:
Thats just the value of how much buffer space is going to be used on the heap, if our packet sizes are large it would need to be larger if I understand correctly, I just set it to half the size of the UDP packet to be safe

Reply:
Ok a quick comment indicating what you set it to and why would be great.

Reply:
Unfortunately this file is autogenerated, did we want to start a doc explaining all the cube settings at some point? There also are some choices made for the memory protection unit which need some explaining.

Reply:
Oh yeah. Made https://github.com/UBC-Thunderbots/Software/issues/1192 and assigned to @akhilveeraghanta